src/test/plot/stacked_bar/test_latency_stacked_bar.py

```diff
@@ -143,12 +146,21 @@ def test_invalid_timestamps(self, mocker):
         columns = [ColumnValue('start'), ColumnValue('end')]
         records = create_records(records_raw, columns)
 
-        mocker.patch.object(recodes_objects, 'columns', ['start', 'end'])
-        mocker.patch.object(recodes_objects, 'data', records)
         mocker.patch.object(target_objects, 'column_names', ['start', 'end'])
         mocker.patch.object(target_objects, 'to_records', return_value=records)
 
+        mock_pass = 'caret_analyze.record.records_service.response_time.ResponseTime.'
+        mocker.patch(mock_pass+'to_all_stacked_bar',
+                     return_value=RecordsFactory.create_instance())
+        mocker.patch(mock_pass+'to_best_case_stacked_bar',
+                     return_value=RecordsFactory.create_instance())
+        mocker.patch(mock_pass+'to_worst_case_stacked_bar',
+                     return_value=RecordsFactory.create_instance())
+        mocker.patch(mock_pass+'to_worst_with_external_latency_case_stacked_bar',
+                     return_value=RecordsFactory.create_instance())
```

Comment:
It may be more appropriate to return a `RecordsInterface` mock instead of an empty record. This is because returning an empty record is not the expectation of these functions.

Reply:
Changed to return RecordsInterface mock